src/Tools/dotnet-user-jwts/src/Helpers/ConsoleTable.cs

```diff
@@ -59,25 +57,94 @@ public void Write()
                 .Select(x => x!.ToString()!.Length).Max())
             .ToList();
 
+        var equalColumnLengths = (Console.WindowWidth / _columns.Count);
+
+        for (var i = 0; i < maxColumnLengths.Count; i++)
```

Comment:
I wonder if there is a way to track excess width here for the cases where the column length is less than the equal column lengths. For example, a narrow column can "donate" some of its width so that it can be reused by larger columns.